protobuf-definitions/proto/mlagents/envs/communicator_objects/agent_info_proto.proto

```diff
@@ -3,6 +3,9 @@ syntax = "proto3";
 option csharp_namespace = "MLAgents.CommunicatorObjects";
 package communicator_objects;
 
+message CustomOutput {
```

Comment:
I think both CustomOutput and CustomParameters should live in separate files. Since these are supposed to be modified by the user, they should be separated from the rest of the proto definitions.

Reply:
👍